plasma_framework/contracts/src/exits/payment/controllers/PaymentStartInFlightExit.sol

```diff
@@ -182,19 +185,40 @@ library PaymentStartInFlightExit {
         return outputIds;
     }
 
+    function getOutputGuardHandlersForInputTxs(
```

Comment:
you forgot the return the array here, that might be the issue

Reply:
You are right @boolafish. I spotted that, when I uploaded it here, hence I removed the comment.
I was surprised the compiler didn't raised an error.